src/duration/extractMinMaxInUnits.js

```diff
@@ -1,5 +1,5 @@
 import durationUnits from './durationUnits';
-import splitSecsToUnits from './splitSecsToUnits';
+import splitSecsToUnits from '../date/splitSecsToUnits';
```

Comment:
I meant let's have

`lib/object/...`
'lib/duration/...`
`lib/...`

Would this get benefits of both approaches?